### DISCUSSION TITLE
[Enhancement] [date-picker] DateTimePicker 组件没有一次性选择的设置.

### DISCUSSION THREAD

**Discussion by iqfu321** [element-plus/element-plus#10523] [Feature Request]
Feature type: **`Enhancement`**

## Details

- Enhanced component name: `date-picker`

## Description

用户在选中日期后,还要点一下确认,很麻烦.

### Additional comments

_(empty)_

**Comment by chenxch** [comment]
This is due to the existence of the time option, because it involves the selection of time, and each click cannot be judged as a confirmation operation.

这个是由于时间选项的存在，因为涉及到了时间的选择，不能将每次点击判定为确认的操作哦。
